refactor(cli): remove show requests and show results (#1049)
IBM/ado: website/docs/migration/1x-to-2x.md

# Migrating from ado 1.x to ado 2.x

ado 2.x introduces a set of breaking changes that remove obsolete commands and
APIs. There are no deprecation warnings or backwards-compatibility shims — the
removed functionality no longer exists at runtime. This guide describes each
breaking change and shows how to update your workflows.

> 

Reviewer: ```suggestion
ado 2.x introduces a set of breaking changes that remove obsolete commands and
APIs. This guide describes each
breaking change and shows how to update your workflows.
```

## Breaking Changes

### Removed: `ado show requests` and `ado show results`

The `ado show requests` and `ado show results` commands have been removed.

> They displayed `MeasurementRequest` and `MeasurementResult` metadata for an

Reviewer: ```suggestion
They displayed `MeasurementRequest` and `MeasurementResult` metadata for an explore
```

operation in separate views. The `ado show trace` command supersedes both: it
provides the same information in a single, unified view with additional

> capabilities such as entity unrolling, field filtering, and YAML output.

Reviewer: ```suggestion
capabilities such as field filtering, and YAML output.
```
Provide same capabilities is == entity unrolling i.e. its not additional

#### Before (ado 1.x)

Inspect measurement requests for an operation:

```shell
ado show requests operation randomwalk-0.5.0-123abc -o csv --output-file requests.csv
```

> Inspect measurement results for an operation:

Reviewer: ```suggestion
Inspect measurement results metadata for an operation:
```

```shell
ado show results operation randomwalk-0.5.0-123abc -o csv --output-file results.csv
```

#### After (ado 2.x)

> Use `ado show trace` to inspect both requests and results in a single command:

Reviewer: ```suggestion
Use `ado show trace` to inspect the trace of measurement requests and optionally metadata about the individual entity measurements made (the result metadata)
```
